### PR TITLE
CLOUDP-126266: Create command 'atlas project settings update'

### PR DESCRIPTION
**Pull request by Razeer123**
## Proposed changes

I added `atlas project settings update` command and tests that should verify the correctness of the implementation.

_Jira ticket:_ [CLOUDP-126266](https://jira.mongodb.org/browse/CLOUDP-126266)

## Command syntax and output

```
michal.derej@M-L7H6RDJWQX mongodb-atlas-cli % ./bin/atlas projects settings describe -P dev                                         
COLLECT DATABASE SPECIFICS STATISTICS ENABLED   DATA EXPLORER ENABLED   PERFORMANCE ADVISOR ENABLED   REALTIME PERFORMANCE PANEL ENABLED   SCHEMA ADVISOR ENABLED
true                                            true                    true                          true                                 true
michal.derej@M-L7H6RDJWQX mongodb-atlas-cli % ./bin/atlas projects settings update --disableCollectDatabaseSpecificsStatistics -P dev
Project settings updated.
michal.derej@M-L7H6RDJWQX mongodb-atlas-cli % ./bin/atlas projects settings describe -P dev                                          
COLLECT DATABASE SPECIFICS STATISTICS ENABLED   DATA EXPLORER ENABLED   PERFORMANCE ADVISOR ENABLED   REALTIME PERFORMANCE PANEL ENABLED   SCHEMA ADVISOR ENABLED
false                                           true                    true                          true                                 true
```

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by fmenezes — DISMISSED** [review]
code LGTM great job 👍

File: docs/atlascli/command/atlas-projects-settings-update.txt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --disableCollectDatabaseSpecificsStatistics
```
Comment (sarahsimpers) [inline docs/atlascli/command/atlas-projects-settings-update.txt:35]: Question: Should this be without the "s" at the end of "specific"?
```suggestion
   * - --disableCollectDatabaseSpecificStatistics
```

File: internal/cli/iam/projects/settings/update.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		return &valueToSet
+	}
+	return nil
+}
+
+// atlas projects(s) settings describe [–-enableCollectDatabaseSpecificsStatistics] [--projectId projectId].
+func UpdateBuilder() *cobra.Command {
+	opts := &UpdateOpts{}
+	cmd := &cobra.Command{
+		Use:     "update",
```
Comment (blva) [inline internal/cli/iam/projects/settings/update.go:94]: nit: add example

File: internal/cli/iam/projects/settings/update_test.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+	"github.com/openlyinc/pointy"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
+	"github.com/mongodb/mongodb-atlas-cli/internal/test"
+	"github.com/stretchr/testify/assert"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
```
Comment (blva) [inline internal/cli/iam/projects/settings/update_test.go:31]:
```suggestion
import (
	"testing"

	"github.com/golang/mock/gomock"
	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
	"github.com/mongodb/mongodb-atlas-cli/internal/test"
	"github.com/openlyinc/pointy"
	"github.com/stretchr/testify/assert"
	atlas "go.mongodb.org/atlas/mongodbatlas"
)
```

File: test/e2e/atlas/project_settings_test.go [hunk 4]
```diff
@@ -21,7 +21,10 @@ import (
 	"os"
 	"testing"
 
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+
 	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	exec "golang.org/x/sys/execabs"
 )
```
Comment (blva) [inline test/e2e/atlas/project_settings_test.go:30]:
```suggestion

	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
	atlas "go.mongodb.org/atlas/mongodbatlas"
	exec "golang.org/x/sys/execabs"
)
```

**Review by blva — DISMISSED** [review]
only nitpicks feel free to add later, great work! 🚀

File: docs/atlascli/command/atlas-projects-settings-update.txt (hunk 1 above)
Comment (Razeer123) [reply docs/atlascli/command/atlas-projects-settings-update.txt:35]: @sarahsimpers In the Scope & Spec docs it's referred as `--disableCollectDatabaseSpecificsStatistics`.

**Comment by Razeer123** [comment]
I added changes requested by @blva. I also updated the docs according to @sarahsimpers hints. Please review the PR again when you have a moment. :)

File: docs/atlascli/command/atlas-projects-settings-update.txt (hunk 1 above)
Comment (blva) [reply docs/atlascli/command/atlas-projects-settings-update.txt:35]: +1 and in the endpoint doc as well https://www.mongodb.com/docs/atlas/reference/api/project-settings-get-one/

**Review by blva — APPROVED** [review]
LGTM, thanks for the changes

**Review by sarahsimpers — APPROVED** [review]
LGTM, thanks!

**Pull request merged**